dcicutils/ff_mocks.py

```diff
@@ -135,37 +135,38 @@ def mocked_s3utils(environments=None, require_sse=False, other_access_key_names=
         with mock.patch.object(ff_utils, "boto3", mock_boto3):
             with mock.patch.object(beanstalk_utils, "boto3", mock_boto3):
                 with mock.patch.object(env_utils, "boto3", mock_boto3):
```

Comment:
The diff here looks complicated but it's just latching onto the wrong thing. Really all that's changed is
```
with mock.patch.object(env_base, "boto3", mock_boto3):
```
became
```
with mock.patch.object(env_basic, "boto3", mock_boto3):
    with mock.patch.object(env_manager, "boto3", mock_boto3):
```
and everything that followed is indented four extra spaces.